### PR TITLE
Fix #2267: RTL Support in ProfileProgress

### LINKED ISSUE #2267: [RTL] High-fi ProfileProgress
`ProfileProgress` needs RTL support as shown in below screen.
UP (User Profile) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by lakshaygarg128**
- Fixes #2267 \High-fi ProfileProgress by adding text alignment of viewstart

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Screenshot_1609273773
Screenshot_1609273815

### CODE REVIEW

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@lakshaygarg128 Need reformatting. I suggest to set up a macro so it will be handy all the time, and it also works with XML files.
Check out wiki here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

Change the title to `Fix #2267: RTL Support in ProfileProgress`

Also add screenshots

**Comment by lakshaygarg128** [comment]
ok @FareesHussain

**Review by FareesHussain — CHANGES_REQUESTED** [review]
This branch is not up-to-date with `develop`

**Review by FareesHussain — APPROVED** [review]

File: app/src/main/res/layout-land/profile_progress_header.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="28sp"
+      android:textAlignment="viewStart"
```
Comment (rt4914) [inline app/src/main/res/layout-land/profile_progress_header.xml]: I don't think this is needed, try to check app without using this parameter.

File: app/src/main/res/layout/profile_progress_header.xml [hunk 2]
```diff
@@ -50,6 +50,7 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="20sp"
+      android:textAlignment="viewStart"
```
Comment (rt4914) [inline app/src/main/res/layout/profile_progress_header.xml]: Ditto

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks for the PR.

There are couple of things which are of concern:
1. Each type of file has 4 xml files but the changes have been made to only some of these files. Why is that? For example `profile_progress_recently_played_story_card.xml` has 4 variations but the changes are in 2 files only.
2. This PR needs to be tested on Tablet device too (Nexus 9). Please add screenshot of the same.
3. The "Admin" written is not aligned with the progress box correctly, check below screenshots.
Screenshot_1609395700
Screenshot_1609395720

File: app/src/main/res/layout-land/profile_progress_header.xml (hunk 1 above)
Comment (lakshaygarg128) [reply app/src/main/res/layout-land/profile_progress_header.xml]: ok

File: app/src/main/res/layout/profile_progress_header.xml (hunk 2 above)
Comment (lakshaygarg128) [reply app/src/main/res/layout/profile_progress_header.xml]: ok

Comment (lakshaygarg128) [reply app/src/main/res/layout/profile_progress_header.xml]: test

Can you pl. clarify that is it totally  correct  ?

Comment (rt4914) [reply app/src/main/res/layout/profile_progress_header.xml]: @lakshaygarg128 Yes, it looks good.

File: app/src/main/res/layout-land/profile_progress_header.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/profile_progress_header.xml]: @lakshaygarg128 You have replied done, but its not reverted?

**Review by rt4914 — CHANGES_REQUESTED** [review]
@lakshaygarg128 PTAL at comments.

File: app/src/main/res/layout-land/profile_progress_header.xml (hunk 1 above)
Comment (lakshaygarg128) [reply app/src/main/res/layout-land/profile_progress_header.xml]: @rt4914  Sorry Sir , I have a doubt while  testing all the data saved as String is working properly in rtl support but the username that is from backend is not working properly in rtl support . Although the alignment in landscape is done .
Here the Admin is not same like profile .


test

**Comment by lakshaygarg128** [comment]
@rt4914 pl. review 
nexus land
simple port
test
nexus port

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**